inhouse_bot/database_orm/tables/game_participant.py

```diff
@@ -66,25 +67,19 @@ class GameParticipant(bot_declarative_base):
             (player_id, player_server_id, role),
             (PlayerRating.player_id, PlayerRating.player_server_id, PlayerRating.role),
         ),
-        {},
     )
 
     # Conservative rating for MMR display
     @hybrid_property
     def mmr(self):
         return 20 * (self.trueskill_mu - 3 * self.trueskill_sigma + 25)
 
-    @hybrid_property
-    def short_name(self):
```

Comment:
I dont think you can remove this short_name helper, if you look at game.py line 84/85

```
    def __str__(self):
        return tabulate(
            {
                "BLUE": [p.short_name for p in self.teams.BLUE],
                "RED": [p.short_name for p in self.teams.BLUE],
            },
            headers="keys",
        )
``` 
This is using the gameparticipant short_name helper function that is used here

Reply:
Its not gonna throw an error immediately but if someone calls __str__ on a game class obj its gonna throw an error, I believe

Reply:
Lol some of my changes from removing all the names were mixed in here. Fixing this